cfme/tests/containers/test_containers_summary.py

```diff
@@ -40,14 +47,18 @@ def test_containers_summary_objects(provider, soft_assert):
            * Goes to Containers summary page and checks how many objects are shown there.
            * Checks the amount is equal
        """
-    container_object = version.pick(objects_key)
-    prov_ui_values, status_box_values = dict(), dict()
     navigate_to(ContainersOverview, 'All')
-    for obj_type in container_object:
-        status_box_values[obj_type] = StatusBox(obj_type).value()
-    for obj_type in container_object:
-        prov_ui_values[obj_type] = getattr(provider.summary.relationships, obj_type).value
-        soft_assert(status_box_values[obj_type] == prov_ui_values[obj_type],
+    # We should wait ~2 seconds for the StatusBox population
+    wait_for(lambda: status_boxs[ContainersProvider].value(), num_sec=10, delay=1)
+    # Collecting status boxs values:
+    status_box_values = {}
+    for obj, sb in status_boxs.items():
+        status_box_values[obj] = sb.value()
```

Comment:
**OPTIONAL**: For such a small loop as this a comprehension may be better
```status_box_values = {obj: sb.value() for obj, sb in status_boxs.items()}```
Replaces 3 lines with just one, which is equally as readable